index.html

```diff
@@ -97,14 +105,15 @@ <h2>Partner</h2>
                     <img class="amazon" src="img/microverse.png" alt="amazon-image">
                     <img class="airbnb" src="img/behance.png" alt="airbnb-image">
                 </div>
-            </div>
-            <div class="mobile-about-footer">
-                <img src="img/v.jpg" alt="cc-logo">
-                <img src="img/mozilla.png" alt="mozilla-image">
-            </div>
-            
+            </div>     
         </section>
     </main>
+    <footer>
+        <div class="mobile-about-footer">
+            <img src="img/mozilla.png" alt="mozilla-image">
+            <img src="img/v.jpg" alt="cc-logo">
+        </div>
+    </footer>
```

Comment:
- [ ] The footer of the home page doesn't appear on a mobile screen. Please fix that as asked by the previous reviewer. 😄 🙏 
_I know that the footer doesn't appear on the image of the original design. However, it's just a presentational image and it's still expected to see a footer on the home page._ 😄

Reply:
Hello @MahmoudBakr23,
![footer](https://user-images.githubusercontent.com/58692590/132938947-c7b8e610-17a8-40ea-8260-1f536d96d1a7.png)

Here is my footer for the mobile homepage. Kindly check it.

Reply:
Hi @Whoistolu 
- [ ] Kindly note that this is not the footer, first, the footer should have a `<footer>` tag element, and second, this part in the image belongs to the "Feature Speakers" section and not the footer 👍 So, please let's work on adding the footer as the same one in the desktop version, please see more details in the review summary 😄 🙏 

**Footer YOUR DESIGN (desktop version) should exist with a white background in the (mobile version)**
![Screenshot (1)1](https://user-images.githubusercontent.com/47625407/132941199-d6ce5262-c688-4eca-81d1-990ab5d707e9.png)